### PR TITLE
feat: Add a toggle between hex and base64

### PR DESCRIPTION
**Pull request by gvking**
A lot less rewrite -> same feature of toggle. Also makes use of session storage instead of chrome storage so we don't request extra permissions from the users, and is more lightweight and easier to access.

### CODE REVIEW

File: log-window.js [hunk 1]
```diff
@@ -186,12 +215,21 @@ class EmeLoggerWindow {
...
           format += ' ' + '[\n';
-          while (data.length) {
-            const row = data.splice(0, 16);
-            format += indentation + '  ';
-            format += row.map(this.byteToHex_).join(', ');
-            format += ',\n';
-          }
+          if(this.toggleStoredInSessionStorage_() == "hex"){
+              while (data.length) {
+                  console.log(data);
```
Comment (beback4u) [inline log-window.js:220]: nit: Can we remove this if it was only for debugging purpose?

File: log-window.js [hunk 2]
```diff
@@ -158,6 +170,23 @@ class EmeLoggerWindow {
     return '0x' + byte.toString(16).padStart(2, '0');
   }
 
+  /**
+   * @param {number} byte
+   * @return {string}
+   * @private
+   */
+  bytesToBase64_(bytes) {
```
Comment (beback4u) [inline log-window.js:178]: I don't see any unit test for the existing `byteToHex_` and for this new `bytesToBase64_` in /spec/log-window-tests.js. Can we add new unit tests for them?

File: log-window.js [hunk 3]
```diff
@@ -158,6 +170,23 @@ class EmeLoggerWindow {
...
+    return btoa(String.fromCharCode.apply(null,new Uint8Array(bytes)));
```
Comment (beback4u) [inline log-window.js:179]: nit: Reformating seems to be needed for this file (i.e., space after comma). You may want to set up automatic running clang-format on Save if you are using VSCode.

**Review by beback4u — COMMENTED** [review]
You will follow the release process: [go/eme-logger-formatter-release](http://go/eme-logger-formatter-release).

1. Do integration tests before landing this change. See [Test by developer](http://go/eme-logger-formatter-release#bookmark=id.y9280drm9wv3) section.
2. Update after landing this change. Find the test tracker link from [Test trackers](http://go/eme-logger-formatter-release#bookmark=id.n13z2jhtoxox) section. Add new test steps for hex<->base64 toggle for "1.1.1 - EME Logger Function Tests".
3. Follow the publishing process for [EME Logger](http://go/eme-logger-formatter-release#bookmark=id.6vludfa7f4gr). Note that the `release-please` branch by automated release tool will pick your change up once landed, I think.

File: log-window.js (hunk 2 above)
Comment (gvking) [reply log-window.js:174]: done.

File: log-window.js (hunk 3 above)
Comment (gvking) [reply log-window.js]: done.

File: spec/log-window-tests.js [hunk 4]
```diff
@@ -312,4 +367,24 @@ describe('Log window', () => {
 
     return obj;
   }
+
+  // These functions are private in log-window.js so we have to copy them to use
```
[the remark this answers is not in the document]
Comment (gvking) [reply spec/log-window-tests.js]: done.

**Review by beback4u — APPROVED** [review]
LGTM

**Pull request merged**